### ISSUE TITLE
ProbCoherence - one mean() too much?

### ISSUE THREAD

**Issue by manuelbickel** [TommyJones/textmineR#37]
Hi Tommy,
with reference to #35 I have another question concerning calculation of coherence. To check my LDA models I want to use different coherence measures and just realized that results from my implementation of the difference measure differs from yours. Maybe I have understood something wrong, in this case, I would appreciate, if you could provide an explanation for better understanding. In the lines highlighted below you use `mean()` in each step of `sapply()` when making calculations on combinations of wi/wj. Would the `mean()` not only have be applied on the full set of probability differences? I have edited your implementation and added comments to highlight what I mean.

```
result <- sapply(1:(ncol(count.mat) - 1), function(x) {
  #  mean(   #this is the mean that I think is too much 
      p.mat[x, (x + 1):ncol(p.mat)]/p.mat[x, x] - Matrix::diag(p.mat)[(x +
                                                                            1):ncol(p.mat)]
    #  , na.rm = TRUE)
  })
  mean(unlist(result), na.rm = TRUE) #instead I would use the mean on all unlisted results
```

**Comment by TommyJones** [comment]
Good catch. I will test the new implementation.

**Comment by TommyJones** [comment]
Fixed with release of [2.1.0](https://github.com/TommyJones/textmineR/releases/tag/v2.1.0)
